src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/get_system_info_response.cc

```diff
@@ -73,6 +75,10 @@ void GetSystemInfoResponse::Run() {
       info.ccpu_version, info.wers_country_code, info.language);
 
   hmi_capabilities_.OnSoftwareVersionReceived(info.ccpu_version);
+  if (!info.hardware_version.empty()) {
+    policy_handler_.OnHardwareVersionReceived(info.hardware_version);
+    hmi_capabilities_.set_hardware_version(info.hardware_version);
+  }
 }
```

Comment:
@AKalinich-Luxoft This block is moved before in the next PR (https://github.com/LuxoftSDL/sdl_core/pull/170/commits/828c645b8c3570850fd9fe80d7e3857954c16720#diff-cb5f9454d22515d0948a6ae79bdb7eb3ccaac3fd01134458bbc66a068a56642cL82), so I prefer to left it here as is to avoid merge conflicts